Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Wildhammer Facial Tattoos

## Changelog:
- Added **8** facial tattoos for Wildhammer Dwarves:
  - `facial_wildhammer_tattoo_01` ("Aerie")
  - `facial_wildhammer_tattoo_02` ("Hinterlands")
  - `facial_wildhammer_tattoo_03` ("Sky"), **disabled for female dwarves**,
  - `facial_wildhammer_tattoo_04` ("Thunder") **disabled for female dwarves**,
  - `facial_wildhammer_tattoo_05` ("Highlands")
  - `facial_wildhammer_tattoo_05_f` ("Northeron")
  - `facial_wildhammer_tattoo_06` ("Gryphon") **disabled for female dwarves**,
  - `facial_wildhammer_tattoo_06_f` ("Batol")
- Added localization to all of the markings. 
- Gave **Kurdran** the "Thunder" marking and **Falstad** the "Gryphon" marking.
  - Also disabled headgear for both characters.

## Images:

<details><summary>Click to expand</summary>

**Aerie** (`facial_wildhammer_tattoo_01`)
![image](https://user-images.githubusercontent.com/78575425/133553483-396b4eaf-b272-47fa-9c93-d9178d688c0f.png)
![image](https://user-images.githubusercontent.com/78575425/133409528-0545ef80-27bd-48fe-931b-601eee6e2293.png)

**Hinterlands** (`facial_wildhammer_tattoo_02`)
![image](https://user-images.githubusercontent.com/78575425/133553495-0051326d-15d9-4bad-82d5-ff8023989467.png)
![image](https://user-images.githubusercontent.com/78575425/133409591-c58502b3-0ffc-4ad7-816a-80e44a9c0ce0.png)

**Sky** (`facial_wildhammer_tattoo_03`)
![image](https://user-images.githubusercontent.com/78575425/133555051-2e2ab6a9-fc3e-44de-8a31-249178a18425.png)
![image](https://user-images.githubusercontent.com/78575425/133409901-e62bcd1b-f818-499c-a4d1-1587cc492731.png)

**Thunder** (`facial_wildhammer_tattoo_04`)
![image](https://user-images.githubusercontent.com/78575425/133553530-28f7f093-c7dc-4b67-9a71-af0d9666a4f4.png)
![image](https://user-images.githubusercontent.com/78575425/133410096-63311cb2-5409-4ca4-90e7-18553083f6dd.png)

**Highlands** (`facial_wildhammer_tattoo_05`)
![image](https://user-images.githubusercontent.com/78575425/133553958-718478d2-538a-4f6a-b3f7-7dc8041c7329.png)
![image](https://user-images.githubusercontent.com/78575425/133414301-d90d218a-b915-46d2-906f-7b3cdd42f7ba.png)

**Northeron** (`facial_wildhammer_tattoo_05_f`)
![image](https://user-images.githubusercontent.com/78575425/133553898-0183f634-7648-4f53-9081-384d8e358d14.png)
![image](https://user-images.githubusercontent.com/78575425/133553664-c06dfa75-fe2a-489a-86ae-542e6e792fc2.png)

**Gryphon** (`facial_wildhammer_tattoo_06`)
![image](https://user-images.githubusercontent.com/78575425/133553832-0d0513ce-67e0-44f1-8b8d-def21caab8ee.png)
![image](https://user-images.githubusercontent.com/78575425/133553737-47ca848e-5ca2-43a0-a3a0-348ff437176e.png)

**Batol** (`facial_wildhammer_tattoo_06_f`)
![image](https://user-images.githubusercontent.com/78575425/133553872-0aa9de6b-0380-49d2-b40e-1735736236e7.png)
![image](https://user-images.githubusercontent.com/78575425/133553631-90b2937b-46d5-4b62-bb31-e8b185c03716.png)

</details>

## Known issues:
- ~~Characters generated with `facial_wildhammer_tattoo_03` or `facial_wildhammer_tattoo_04` should be bald for their tattoos to show fully.~~
  - Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/commit/9e2f0aa997b13100e94cb0468d3bd8a11b73eeb3.
- Some markings will appear to have a line going through the top middle of the character's head. This is an issue with CK3, not an issue with the markings. If you zoom in on the character, the line disappears.

<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/133555323-ef2477e6-7a7b-49af-bcf8-e59e33e1deb7.png)
![image](https://user-images.githubusercontent.com/78575425/133555350-02660adf-c59d-4536-a813-7e449f113276.png)

</details>

## Tasks:
- [x] Add the remaining male facial tattoos:
  - [x] Add `facial_wildhammer_tattoo_05` ("Highlands").
  - [x] Add `facial_wildhammer_tattoo_06` ("Gryphon").
  - [ ] _(Optional)_ Add `facial_wildhammer_tattoo_07` ("Ironforge").
    - Based on the name, maybe this was intended to be a Bronzebeard tattoo, but we could still use it for Wildhammer Dwarves.
- [x] Add female facial tattoos:
  - [x] Add `facial_wildhammer_tattoo_05_f` ("Highlands").
  - [x] Add `facial_wildhammer_tattoo_06_f` ("Gryphon").
- [x] Localize the tattoos.
- [x] Wildhammer Dwarves with prominent head tattoos should be made bald (see **Tasks**).


## Tests:


# How to test:

There are a few ways to test these markings:

**A**. Open up the game in [Debug Mode](https://ck3.paradoxwikis.com/Console_commands#Enabling_debug_mode) and view them in the **Portrait Editor** under `gene_facial_markings`.
**B**. View the markings in **Ruler Designer**.
**C**. Start a game and look at characters with the **_Wildhammer_** culture to see how the markings look in-game. **Some dwarves have no markings, this is intended and normal.**

Provide feedback on the look of the markings and the localization used for their names (all names except Batol and Northeron are taken directly from the names they used for the markings in WoW).
